client/src/components/Intro/index.tsx

```diff
@@ -16,6 +16,8 @@ interface IntroProps {
   pending?: boolean;
   slug?: string;
   username?: string;
+  onDonationAlertClick?: () => void;
```

Comment:
```suggestion
  onDonationAlertClick: () => void;
```
It's always present.